package.json

```diff
@@ -16,11 +16,13 @@
     "prettier-plugin-solidity": "^1.0.0-beta.2 ",
     "solhint": "^3.3.4",
     "solhint-config-keep": "github:keep-network/solhint-config-keep",
+    "solidity-coverage": "^0.7.16",
     "typescript": "^3.7.0"
   },
   "name": "@keep-network/coverage-pool",
   "scripts": {
     "build": "hardhat compile",
+    "coverage": "NODE_ENV=system-test hardhat coverage",
```

Comment:
If we manage to fix coverage for system tests, then how about splitting coverage to unit & system, ie
```
    "test:coverage": "yarn test:unit:coverage && yarn test:system:coverage",
    "test:unit:coverage": "hardhat coverage",
    "test:system:coverage": "NODE_ENV=system-test hardhat coverage --testfiles './test/system/*.test.js'",
```

Reply:
We can do that, but I'm not sure what happens if we run coverage for unit tests and system tests separately (`hardhat coverage` called twice). The generated report will probably contain only coverage from system tests (unit test coverage will be overwritten).